plugins/modules/nmcli.py

```diff
@@ -1004,10 +1004,28 @@
             private-key:
                 description: The 256 bit private-key in base64 encoding.
                 type: str
+                no_log: True
```

Comment:
There is no `no_log` field in the module documentation. You can only use that in the argument spec, but since `wireguard` has no sub-argument spec, it's not possible to supply that at the moment.
```suggestion
```